apps/dashboard/config/routes.rb

```diff
@@ -98,7 +98,7 @@
     delete '/activejobs' => 'active_jobs#delete_job', :as => 'delete_job'
   end
 
-  get '/jobs/info/:cluster/:id' => 'jobs#info', :defaults => { :format => 'json' }, :as => 'jobs_info'
+  get '/jobs/info/:cluster/:id' => 'jobs#pm_job_details', :defaults => { :format => 'turbo_stream' }, :as => 'pm_job_details'
```

Comment:
I named this route `jobs#pm_job_details` just because I don't know if these partials are going to be useful for other pages. So we may as well make the API specific for the project manager.